__fixtures__/test-project/web/src/scaffold.css

```diff
@@ -165,6 +175,9 @@
 .rw-table td {
   @apply bg-white text-gray-900;
 }
+.rw-table tr:nth-child(odd) td {
```

Comment:
@Tobbe 
Just find another issue, I am not sure what was the intended design but I feel like it would be better to change the backgrounds of ths in tbody.

Other than this issuem  it looks good to me. ✅

Reply:
I agree. That doesn't look good. Is it only a problem for the pure css styles, or for TW too?

I'm also not entirely sure what the intended design is/was. I'll do a little digging.

Reply:
This is what it's supposed to look like

![image](https://user-images.githubusercontent.com/30793/163863505-f5d8f479-0b6d-4cad-bcd4-b52d461b33a0.png)

Reply:
@Tobbe I have updated branch in my forked repo however I am not sure how to push those changes to this pr to be honest.

Reply:
@ccnklc You figured it out, right? This is the commit you wanted? "color th s in tbody as well"